packages/next/src/server/app-render/create-server-components-renderer.tsx

```diff
@@ -33,9 +33,9 @@ export function createServerComponentRenderer<Props>(
 ): (props: Props) => JSX.Element {
   // We need to expose the `__webpack_require__` API globally for
   // react-server-dom-webpack. This is a hack until we find a better way.
-  if (ComponentMod.__next_app_webpack_require__) {
+  if (ComponentMod.__next_app_require__) {
```

Comment:
If we're changing this at all in this PR, let's make it  `ComponentMod.__next_app__ = { require, loadChunk }`, as I need this as well for next build.